src/ui/inspector/Inspector.vue

```diff
@@ -1,22 +1,66 @@
 <template>
```

Comment:
This is a nice way of handling an arbitrary number of tabs. For right now though, let's assume that there will only be two tabs - properties and styles if it simplifies the code. I'm not convinced that tabs are scaleable in the inspector, so it's not something we want to generalize just yet.

Reply:
Done.